libsource/src/main/java/com/virtusize/libsource/network/VirtusizeApiTask.kt

```diff
@@ -189,28 +248,39 @@ internal class VirtusizeApiTask(
     /**
      * Check if the response of the API request is a JSON array
      * @param apiRequestUrl The input stream of bytes
+     * @return the boolean value to tell whether the response of the apiRequestUrl is a JSON array.
      */
     private fun responseIsJsonArray(apiRequestUrl: String): Boolean {
         return apiRequestUrl.contains(VirtusizeEndpoint.ProductType.getPath())
                 || apiRequestUrl.contains(VirtusizeEndpoint.UserProducts.getPath())
     }
 
     /**
-     * Returns the contents of an InputStream as a String.
+     * Returns the contents of an [InputStream] as a String.
      * @param inputStream The input stream of bytes
+     * @return the string from scanning through the inputStream
      */
     private fun readInputStreamAsString(inputStream: InputStream): String? {
         val scanner: Scanner = Scanner(inputStream).useDelimiter("\\A")
         return if (scanner.hasNext()) scanner.next() else null
     }
 
     /**
-     * Returns the VirtusizeError that is associated with an API error
+     * Returns the [VirtusizeError] that is associated with an API error
      * @param urlPath The endpoint path of an API request
      * @param responseCode The response code of an API request
      * @param response The response from an API request
+     * @return the network related [VirtusizeError]
      */
-    private fun virtusizeNetworkError(urlPath: String?, responseCode: Int?, response: Any?): VirtusizeError {
-        return VirtusizeError(VirtusizeErrorType.NetworkError, responseCode, "$urlPath - ${response?.toString()}")
+    private fun virtusizeAPIError(
```

Comment:
@akueisara
It would be great to have consistency on naming convention like `variables -> noun` and `functions -> verb`.
How do you think for function naming `virtusizeAPIError -> getVirtusizeAPIerror`?
Or Creating VirtusizeAPIError class and create new constructor would look nice!
It doesn't need to be modified for this on this PR.

Reply:
@gpark-vs
Yeah it's a good idea!
However, after I looked into it, I decided to use the extension function VirtusizeErrorType.virtusizeError in VirtusizeErrorType.kt to create a VirtusizeError object related to the API error.
So the original function is now changed to getAPIErrorMessage to only handle the message to be passed into VirtusizeError. Since the function getAPIErrorMessage is only used in this VirtusizeApiTask class, I think it would be good to keep it here.

Reply:
Yep, that would be enough! Thanks Kuei 🙏